components/eventing-controller/controllers/subscription/nats/reconciler_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"testing"
 	"time"
 
-	utils "github.com/kyma-project/kyma/components/eventing-controller/controllers/subscription"
+	utils "github.com/kyma-project/kyma/components/eventing-controller/controllers/subscription/testing"
```

Comment:
nit: is the import alias still required ? In theory we can access the functions via `testing.FunctionName` now

Reply:
That's right! Will change it 👍